nacl-fast.js

```diff
@@ -2361,17 +2361,16 @@ nacl.setPRNG = function(fn) {
   // Initialize PRNG if environment provides CSPRNG.
   // If not, methods calling randombytes will throw.
   var crypto;
-  if (typeof window !== 'undefined') {
-    // Browser.
-    if (window.crypto && window.crypto.getRandomValues) {
-      crypto = window.crypto; // Standard
-    } else if (window.msCrypto && window.msCrypto.getRandomValues) {
-      crypto = window.msCrypto; // Internet Explorer 11+
-    }
-    if (crypto) {
+  if (typeof process === 'undefined' || process.browser) {
```

Comment:
I think actually, you're right about `process.browser`, let's just do the normal detection of what I intended — if we're asking if `self.crypto` exists, let's check that, and try to fallback to require. I'll send pull request soon and post a link here.